packages/yarnpkg-doctor/sources/cli.ts

```diff
@@ -284,6 +289,10 @@ async function processWorkspace(workspace: Workspace, {configuration, fileList,
   const progress = StreamReport.progressViaCounter(fileList.length + 1);
   const reportedProgress = report.reportProgress(progress);
 
+  for (const scriptName of workspace.manifest.scripts.keys())
+    if (scriptName.match(/^(pre|post)/))
```

Comment:
this leaves `postinstall` (and other documented lifecycle scripts) alone, right?

Reply:
Oh right - yes, `preinstall` and `postinstall` are basically aliases for `install` atm. I'll update the check 👍